frame/contracts/src/chain_extension.rs

```diff
@@ -111,7 +117,7 @@ pub trait ChainExtension<C: Config> {
 	/// In case of `Err` the contract execution is immediately suspended and the passed error
 	/// is returned to the caller. Otherwise the value of [`RetVal`] determines the exit
 	/// behaviour.
-	fn call<E>(func_id: u32, env: Environment<E, InitState>) -> Result<RetVal>
+	fn call<E>(&mut self, func_id: u32, env: Environment<E, InitState>) -> Result<RetVal>
```

Comment:
Since it's a breaking change, how about adding it to https://github.com/paritytech/substrate/blob/master/frame/contracts/CHANGELOG.md ?

Reply:
Changelog is out of date. I need to redo it and use polkadot releases instead. I will do this in a follow up.